### PR TITLE
Stir the new behaviours in

### PR DESCRIPTION
**Pull request by notwaldorf**
This just adds the basic state behaviours in. I will have a followup CL to add the focus ones.

### CODE REVIEW

**Comment by notwaldorf** [comment]
@cdata PTAL

File: paper-checkbox.html [hunk 1]
```diff
@@ -78,6 +79,10 @@
     // The custom properties shim is currently an opt-in feature.
     enableCustomStyleProperties: true,
 
+    behaviors: [
+      Polymer.PaperButtonBehavior
```
Comment (cdata) [inline paper-checkbox.html]: Woot :deciduous_tree:

File: paper-checkbox.html [hunk 2]
```diff
@@ -136,6 +138,17 @@
...
     },
 
+    // button-behavior hook
+    _buttonStateChanged: function() {
+      this.checked = this.active;
+    },
+
+    _checkedChanged: function(checked) {
+      this.setAttribute('aria-checked', this.checked ? 'true' : 'false');
+      this.active = this.checked;
```
Comment (cdata) [inline paper-checkbox.html]: Are `active` and `checked` synonymous? We've gone back and forth on this a bit, so I'm just curious if we settled on something.

**Comment by cdata** [comment]
Non-blocking question, otherwise LGTM :+1:

File: paper-checkbox.html (hunk 2 above)
Comment (notwaldorf) [inline paper-checkbox.html]: I think so. This is also how @sjmiles added the behaviour to `paper-toggle-button`. 

I think normally `active` would be enough, but we (the internet) are used to setting `checked` on a checkbox, so it's basically making sure that checked iff active.

**Pull request merged**